.nfpm_template.yaml

```diff
@@ -0,0 +1,20 @@
+# nfpm configuration file used as template.
+# all "IG_" will be replaced by proper values in the release pipeline.
+name: "ig"
+arch: "IG_ARCH"
+platform: "linux"
+version: "IG_VERSION"
+section: "devel"
+replaces:
+- ig
+provides:
+- ig
+maintainer: "Francis Laniel <flaniel@linux.microsoft.com>"
```

Comment:
This is the package's maintainer that people will see when inspecting the .deb/.rpm files. Shouldn't it be something like `The Inspektor Gadget authors`?